src/qt/governancelist.cpp

```diff
@@ -387,8 +387,11 @@ void GovernanceList::showProposalContextMenu(const QPoint& pos)
         return;
     }
 
+    QString proposal_url = proposal->url();
+    proposal_url.replace(QChar('&'), QString("&&"));
+
     // right click menu with option to open proposal url
-    QAction* openProposalUrl = new QAction(proposal->url(), this);
+    QAction* openProposalUrl = new QAction(proposal_url, this);
```

Comment:
I don't understand the purpose of c36bb8e6fba143a0fbadb1599c5c5360de7c9181

Reply:
same as https://github.com/bitcoin-core/gui/pull/828

(see screenshot)